package.json

```diff
@@ -41,14 +41,13 @@
   },
   "homepage": "https://github.com/solid/solid-permissions",
   "dependencies": {
-    "babel-cli": "^6.14.0",
-    "babel-preset-es2015": "^6.14.0",
-    "shorthash": "0.0.2",
-    "solid-namespace": "0.0.1"
+    "solid-namespace": "0.1.0"
   },
   "devDependencies": {
-    "rdflib": "^0.10.0",
-    "solid-web-client": "0.0.5",
+    "babel-cli": "^6.14.0",
+    "babel-preset-es2015": "^6.14.0",
+    "rdflib": "^0.12.1",
+    "solid-web-client": "0.0.8",
```

Comment:
Why is solid-web-client in the `devDependencies`? Shouldn't it be in the `dependencies` hash?